source/install/desktop.rst

```diff
@@ -59,6 +59,10 @@ Linux (Beta)
 
 3. If you need a Desktop Entry for the application to integrate into a desktop environment, please refer to https://wiki.archlinux.org/index.php/Desktop_entries
 
+To install the snap on [snap supported](https://snapcraft.io/docs/core/install) systems from the store.
+
+    snap install mattermost-desktop --beta
```

Comment:
Do you need sudo for this? The example on https://snapcraft.io/ uses sudo and I was just wondering if it's required for the desktop app.